todo_app/api/trelloIds.py

```diff
@@ -0,0 +1,5 @@
+trello_board_id = "65d6337573a6275aed0ec68a"
```

Comment:
If/when we switch these to environment variables, it's worth noting that the simplest way (`trello_board_id = os.getenv("TRELLO_BOARD_ID")` will work nicely for now _but_ values may not be set as you expect for tests:

The reason it could be a problem is that these lines of code run when this file is first imported. That works perfectly right now, but:

* When we introduce tests we'll sometimes want to do some setup before we start the app
* This can include loading fake/test environment variables
* But these variables will already have had their values assigned by the time we get to our setup, so we won't affect them

We can fix that by messing with our import order carefully, but we typically end up with very fragile code, so a better solution is likely to convert these to a function, like how you handle the API Key and Token